changelog/IcspTTolTAeaYCJWYDikYA.md

```diff
@@ -0,0 +1,3 @@
+audience: general
+level: silent
+---
```

Comment:
Could we delete this changelog?

Reply:
I think we'll get an error in the CI if we do that.

Reply:
Even if we have this one? `changelog/issue-3540.md`

https://github.com/taskcluster/taskcluster/blob/c5e32265fc6d624a141de9d1fd1b4cbd8ab363ec/changelog/issue-3540.md

Reply:
Oh silly me, good point! 😆